src/TestFramework/TestFramework/Assertions/Assert.That.cs

```diff
@@ -10,6 +10,29 @@ namespace Microsoft.VisualStudio.TestTools.UnitTesting;
 /// </summary>
 public static partial class AssertExtensions
 {
+    // Constants for standardized display values
+    private const string FailedToEvaluate = "<Failed to evaluate>";
```

Comment:
Do we want this to be localized?

Reply:
Fine by me. FYI it wasn't localized before.